extension/arquillian-drone-browserstack-extension/README.adoc

```diff
@@ -0,0 +1,138 @@
+= Arquillian Drone BrowserStack extension
+This is an Arquillian extension that enables you to use the BrowserStack as a Drone WebDriver there in your functional tests.
+
+=== How to use it:
+
+1):: First of all, you need to have this extension on your project's classpath. Eg. as a maven dependency:
+[source,xml]
+----
+    <dependency>
+        <groupId>org.jboss.arquillian.extension</groupId>
+        <artifactId>arquillian-drone-browserstack-extension</artifactId>
+        <version>${version.org.jboss.arquillian.drone}</version>
+    </dependency>
+----
+
+2):: To use the BrowserStack driver you need to specify *browserstack* as a *browser* property:
+[source,xml]
+----
+    <extension qualifier="webdriver">
+        <property name="browser">browserstack</property>
+        ...
+    </extension>
+----
+
+3):: You can choose if you want to specify your credentials to access your account or to provide the URL directly with the credentials included.
```

Comment:
BTW. with the arquillian-core 1.1.9.Final and newer it should be possible to specify it on the command line as well: http://arquillian.org/blog/2015/09/02/arquillian-core-1-1-9-Final/